### ISSUE TITLE
Direct use of booleans in conditions

### ISSUE THREAD

**Issue by eugene-sy** [HairyFotr/linter#13]
looks like linter do not find conditions like

``` scala
if (false) "not linted"
if (true) { "not linted" } else { "not linted" }
```

Here should be a warning, that conditional expression should be omitted.

My stack is:
Scala 2.10.2, sbt 0.13.0, Play 2.2.1
Linter is added as jar and scalacOpt to Build.scala file.
